script.js

```diff
@@ -1,92 +1,138 @@
-$(document).ready(() => {
-
-});
 
+const { jsPDF } = window.jspdf;
 
-var creditPoints = [];
 
-$('#submitNum').click(() => {
-    // get number of  courses from input
-    var numInput = $('#num').val();
+$(document).ready(() => {
 
-    // convert number of courses to type number
-    var numberOfCourses = Number(numInput);
-    console.log(numberOfCourses);
+});
 
-    // clear content of div.calculate 
-    document.getElementById('courses-container').innerHTML = '';
 
-    // append number of inputs that corresponds to number of courses to calculate
-    for (var i = 0; i < numberOfCourses; i++) {
-        document.getElementById('courses-container').insertAdjacentHTML("beforeend", `<input class="animated fadeIn" placeholder="COURSE ${i + 1}" type="text">  <input type="number" placeholder="CREDIT UNIT" class="credit-unit animated fadeIn">  <input type="text" placeholder="GRADE" class="grade animated fadeIn"><br/><hr/><br/>`);
-    }
-});
+        var creditPoints = [];
+        var allCreditUnitInputs = []
+        var numberOfCourses
+        var result
+
+        $('#submitNum').click(() => {
+            numInput = $('#num').val();
```

Comment:
what happened to the `var` keyword here.
Made me feel this had been declared else where